src/controllers/userProfileController.js

```diff
@@ -1686,37 +1686,82 @@ const userProfileController = function (UserProfile, Project) {
     });
   };
 
-  const deleteInfringements = async function (req, res) {
-    if (!(await hasPermission(req.body.requestor, 'deleteInfringements'))) {
-      res.status(403).send('You are not authorized to delete blue square');
-      return;
-    }
-    const { userId, blueSquareId } = req.params;
-    // console.log(userId, blueSquareId);
+  // const deleteInfringements = async function (req, res) {
+  //   if (!(await hasPermission(req.body.requestor, 'deleteInfringements'))) {
+  //     res.status(403).send('You are not authorized to delete blue square');
+  //     return;
+  //   }
+  //   const { userId, blueSquareId } = req.params;
+  //   // console.log(userId, blueSquareId);
+
+  //   UserProfile.findById(userId, async (err, record) => {
+  //     if (err || !record) {
+  //       res.status(404).send('No valid records found');
+  //       return;
+  //     }
+
+  //     const originalinfringements = record?.infringements ?? [];
+
+  //     record.infringements = originalinfringements.filter(
+  //       (infringement) => !infringement._id.equals(blueSquareId),
+  //     );
+
+  //     record
+  //       .save()
+  //       .then((results) => {
+  //         userHelper.notifyInfringements(originalinfringements, results.infringements);
+  //         res.status(200).json({
+  //           _id: record._id,
+  //         });
+  //       })
+  //       .catch((error) => res.status(400).send(error));
+  //   });
+  // };
```

Comment:
```suggestion
```
Deleting this old code instead of just commenting it out and duplicating it makes the commit history slightly easier to see/compare changes and reduces clutter from old code.